telco-hub/configuration/reference-crs-kube-compare/optional/logging/clusterLogNS.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-logging
+  annotations:
+    workload.openshift.io/allowed: management
```

Comment:
We do not enable workload partitioning for the hub cluster. This annotation should be optional. 
One option is we can remove it here entirely and add it to the global ignored list.